sdk/common/src/main/java/io/opentelemetry/sdk/common/export/CompletableResultCode.java

```diff
@@ -107,4 +109,33 @@ public CompletableResultCode whenComplete(Runnable action) {
     }
     return this;
   }
+
+  /** Waits until this instances completes, regardless of the result. */
+  public void join() throws InterruptedException {
```

Comment:
I think we can leave out this method, joining without timeout is rarely a good idea.

Reply:
I use it `forceFlush`. There is no good option how long should I wait there. And if you say that `forceFlush` should be async, I will just change my argument to `shutdown` :)